cmd/lakefs/cmd/run.go

```diff
@@ -14,6 +14,7 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/coreos/go-oidc"
```

Comment:
This is v2 - I think you want to use `github.com/coreos/go-oidc/v3/oidc`

Reply:
done